spring-kafka/src/main/java/org/springframework/kafka/listener/CommonErrorHandler.java

```diff
@@ -41,13 +41,25 @@
 	 * When true (default), all remaining records including the failed record are passed
 	 * to the error handler.
 	 * @return false to receive only the failed record.
+	 * @deprecated in favor of {@link #seeksAfterHandling()}.
 	 * @see #handleRecord(Exception, ConsumerRecord, Consumer, MessageListenerContainer)
 	 * @see #handleRemaining(Exception, List, Consumer, MessageListenerContainer)
 	 */
+	@Deprecated
 	default boolean remainingRecords() {
 		return false;
 	}
 
+	/**
+	 * Return true if this error handle performs seeks on the failed record and remaining
```

Comment:
Typo:

```suggestion
	 * Return true if this error handler performs seeks on the failed record and remaining
```

and `@since 2.9`